core/templates/pages/exploration-editor-page/statistics-tab/issues/playthrough-issues.component.html

```diff
@@ -12,7 +12,7 @@
     </div>
     <div class="col-md-9 oppia-issues-border-gray">
       <div ng-repeat="issue in issues">
-        <div ng-if="isIssueOnInitDisplay()" style="display: block;" id="issue<[getIssueIndex(issue)]>">
+        <div ng-if="isIssueOnInitDisplay()" class="oppia-display-block" id="issue<[getIssueIndex(issue)]>">
```

Comment:
Here and below -- you can just use the bootstrap classes instead, right? https://getbootstrap.com/docs/4.0/utilities/display/

Reply:
Actually I don't see why this is necessary at all. The ngIf would handle it for us, right? Let's just get rid of these class entirely.

Reply:
Done.